packages/blade/src/utils/storybook/StoryPageWrapper.tsx

```diff
@@ -101,9 +101,9 @@ const StoryPageWrapper = (props: StoryPageWrapperTypes): React.ReactElement => {
   return (
     <BladeProvider themeTokens={paymentTheme}>
       <WithGlobalStyles>
-        <Title size="xlarge" marginBottom="spacing.3">
+        <Display size="small" marginBottom="spacing.3">
```

Comment:
so title xlarge and large both point to Display small, wouldn't things look visually break for consumers once they migrate?

Reply:
ops nope. find and replace gone wrong. It should be Heading xlarge in other places. Fixed it for this and some other storybook heading size issues